lib_test/messages.ml

```diff
@@ -49,6 +49,11 @@ let files = [
   "query-xen-platform-pv-driver-info.json", Command (None, Query_xen_platform_pv_driver_info);
   "query-xen-platform-pv-driver-info-result.json", Success (None, Xen_platform_pv_driver_info { product_num=3; build_num=1; });
   "query-xen-platform-pv-driver-info-result-error-notanint.json", Error (None, { cls="JSONParsing"; descr="Expected int, got string \"foo\" in {\"return\": {\"product-num\": \"foo\", \"build-num\": 1}}"; });
+  "device_del.json",               Command (None, Device_del "usb1");
+  "device_add_usbcontroller.json", Command (None, Device_add ("usb-ehci", "ehci", None));
+  "device_add_usbdevice.json",     Command (None, Device_add ("usb-host", "usb1", Some ("ehci.0","2","2")));
```

Comment:
Adding a test failing because we did not pass all the arguments would be good, but the tests here are not yet supporting this so I think it's out of scope for this PR

Reply:
yes, I think so.